crates/typst-library/src/text/mod.rs

```diff
@@ -766,35 +790,97 @@ impl PlainText for Packed<TextElem> {
 }
 
 /// A lowercased font family like "arial".
-#[derive(Clone, Eq, PartialEq, Hash)]
-pub struct FontFamily(EcoString);
+#[derive(Debug, Clone)]
+pub struct FontFamily {
+    name: EcoString,
+    coverage: Option<Regex>,
```

Comment:
```suggestion
    covers: Option<Regex>,
```
Brief docs on the fields would be nice.